app/src/main/java/com/simplecity/amp_library/utils/LogUtils.java

```diff
@@ -18,8 +17,7 @@ public static void logException(String tag, String message, @Nullable Throwable
                 throwable.printStackTrace();
             }
         } else {
-            Crashlytics.log(Log.ERROR, tag, message + "\nThrowable: " + (throwable != null ? throwable.getMessage() : null));
```

Comment:
Anywhere where Crashlytics was logging `Log.ERROR`, we should log to `Log.e`

Reply:
Sorry about this. I was very curious about testing the app without crashlytics. Thanks for pointing these out.